src/argument.cpp

```diff
@@ -206,5 +208,18 @@ argument argument::element(std::size_t i) const
     return argument{shape{this->get_shape().type()}, this->data() + offset};
 }
 
+void save_argument(const argument& a, const std::string& filename)
+{
+    migraphx::value v = to_value(a);
+    write_buffer(filename, to_msgpack(v));
```

Comment:
Actually, you could make this one line as `write_buffer(filename, to_msgpack(to_value(a)))`. I dont think its too long.

Reply:
fixed